### PR TITLE
cassie/cassie_v2 sdf: remove first primatic joint, add color information

### PR DESCRIPTION
**Pull request by proyan**

### CODE REVIEW

File: python/example_robot_data/robots_loader.py [hunk 1]
```diff
@@ -161,11 +175,27 @@ def loadANYmal(withArm=None):
 
 class CassieLoader(RobotLoader):
     path = 'cassie_description'
-    sdf_filename = "cassie_v2.sdf"
+    if tuple(int(i) for i in pin.__version__.split('.')) > (2, 9, 1):
+        sdf_filename = "cassie.sdf"
+    else:
+        sdf_filename = "cassie_v2.sdf"
```
Comment (nim65s) [inline python/example_robot_data/robots_loader.py:181]:
```suggestion
    sdf_filename = "cassie.sdf" if tuple(int(i) for i in pin.__version__.split('.')) > (2, 9, 1) else "cassie_v2.sdf"
```

**Comment by nim65s** [comment]
Don't worry about the CI.

If loading cassie is working for you, I think we can merge and continue with the release

File: python/example_robot_data/robots_loader.py (hunk 1 above)
Comment (nim65s) [reply python/example_robot_data/robots_loader.py:181]: (I don't think you can actually execute code in the class declaration, but you can execute code in the parameter declaration… But I might be wrong)

Comment (nim65s) [reply python/example_robot_data/robots_loader.py:181]: my bad, this works, I wasn't aware of it

**Comment by proyan** [comment]
I'm finished with the changes, you could merge it now.
Thanks for your help

**Pull request merged**